### PR TITLE
show version file download errors

### PR DESCRIPTION
**Pull request by fritzmg**
Implements #114 

screen shot 2017-01-10 at 13 08 40

### CODE REVIEW

File: check/i18n/en.php [hunk 1]
```diff
@@ -182,5 +182,8 @@
 	'PHP image processing' => 'PHP image processing',
 	'At least one of the supported image processing libraries is available.' => 'At least one of the supported image processing libraries is available.',
 	'None of the supported image processing libraries are available.' => 'None of the supported image processing libraries are available.',
-	'As of Contao 4.3 one of the supported image processing libraries must be availble (GD, Imagick or Gmagick).' => 'As of Contao 4.3 one of the supported image processing libraries must be availble (GD, Imagick or Gmagick).'
+	'As of Contao 4.3 one of the supported image processing libraries must be availble (GD, Imagick or Gmagick).' => 'As of Contao 4.3 one of the supported image processing libraries must be availble (GD, Imagick or Gmagick).',
```
Comment (xchs) [inline check/i18n/en.php]: Typo: availble -> avail**a**ble

Comment (fritzmg) [inline check/i18n/en.php]: Ah, thx. This is actually a typo of another PR. I'll fix it here though.

**Comment by xchs** [comment]
> Error while retrieving version file: SSL certificate problem: unable to get local issuer certificate

The **two** colons in the error message look a bit odd. Instead of the second colon, we could possibly set a full stop or a hyphen:
```
Error while retrieving version file: SSL certificate problem. Unable to get local issuer certificate.
```
```
Error while retrieving version file: SSL certificate problem - unable to get local issuer certificate
```
WDYT?

**Comment by fritzmg** [comment]
`SSL certificate problem. Unable to get local issuer certificate.` is an error message generated by a component within PHP. The translation string is simply:
```
'Error while retrieving version file: %s.'
```

**Comment by xchs** [comment]
Ah, okay.

**Comment by fritzmg** [comment]
But we could change
```
'Error while retrieving version file: %s.'
```
to
```
'Error while retrieving version file - %s.'
```
then it would say
```
'Error while retrieving version file: SSL certificate problem: Unable to get local issuer certificate.'
```
for example.

**Comment by xchs** [comment]
Beside the above one, are there any other error messages coming directly from PHP that also include a colon in the error message? If so, the "problem" might also occur elsewhere.

**Comment by fritzmg** [comment]
I'll leave the wording as it is, for now.

**Pull request merged**